packages/x-engine/package.json

```diff
@@ -32,5 +29,11 @@
   },
   "devDependencies": {
     "check-engine": "^1.10.1"
+  },
+  "peerDependencies": {
+    "webpack": "^4.42.0"
```

Comment:
**question**: is there a reason why `x-engine` can only support `webpack@4.42.0` and up? you tend to want peer dependencies to be as lenient as possible so you don't cause npm installations to fail.
```suggestion
    "webpack": "4.x"
```